biogeochem/EDCohortDynamicsMod.F90

```diff
@@ -22,6 +22,7 @@ module EDCohortDynamicsMod
   use EDTypesMod            , only : min_n_safemath
   use EDTypesMod            , only : nlevleaf
   use FatesInterfaceMod      , only : hlm_use_planthydro
+  use FatesInterfaceMod      , only : hlm_parteh_model
```

Comment:
Why is it 'hlm_parteh_model' and not just 'parteh_model'? I guess I'm a little confused about whether PARTEH is allied with the hlm or is fully downstream of FATES...

Reply:
The hlm_ tag indicates this was a variable passed in by the HLM, mostly likely from the namelist.   Its just kind of following the convention of the other variables that do the same.  However, I don't know if I'm a big fan of this convention anymore, because it confusing, and I'm not sure it has much meaning where a variable "came from".